dist/index.js

```diff
@@ -972,15 +972,27 @@ var __importStar = (this && this.__importStar) || function (mod) {
 Object.defineProperty(exports, "__esModule", { value: true });
 const core = __importStar(__webpack_require__(470));
 const exec = __importStar(__webpack_require__(986));
+const os = __importStar(__webpack_require__(87));
 class Ndk {
-    constructor(path) {
-        this.path = path;
+    constructor() {
```

Comment:
Because this file is auto-generated, can you delete this file from git and add it to `.gitignore` ?
This would avoid confusions

Reply:
we can remove the `index.js` because the GHA isn't compiling the code it's directly using it from the `dist` directory.